### PR TITLE
libyaml 0.2.5

### PR DESCRIPTION
**Pull request by sevan**

### CODE REVIEW

File: Library/Formula/libyaml.rb [hunk 1]
```diff
@@ -1,28 +1,12 @@
...
-  bottle do
-    cellar :any
-    sha256 "557b32dbf6e6798972e6f9594a91cca044f90f92f410e0eb3ebcbee199f781aa" => :el_capitan
-    sha1 "fe12271b6ad73806e26dd5e1c7d9090c739361a1" => :yosemite
-    sha1 "c1db85f1e26699b0788cea8383fba127cfb4c83b" => :mavericks
-    sha1 "5b2af750962a1cdc36384f49d2fe96b0f00d5fda" => :mountain_lion
-  end
+  url "http://pyyaml.org/download/libyaml/yaml-0.2.5.tar.gz"
+  mirror "https://mirrors.edge.kernel.org/debian/pool/main/liby/libyaml/libyaml_0.2.5.orig.tar.gz"
+  sha256 "0c4e000253ef7187feeb940a01a1c7594f28d63aa16f978e892a0e2864f58614"
```
Comment (mistydemeo) [inline Library/Formula/libyaml.rb:6]: Hm, I'm seeing a different sha256 here: 

```
SHA256: c642ae9b75fee120b2d96c712538bd2cf283228d2337df2cf2988e3c02678ef4
Warning: Formula reports different SHA256: 0c4e000253ef7187feeb940a01a1c7594f28d63aa16f978e892a0e2864f58614
```

Comment (sevan) [reply Library/Formula/libyaml.rb:6]: File on the mirror had a different checksum & and accessing pyyaml.org via http served an html page with the redirect!
Should be fixed now.

Comment (sevan) [reply Library/Formula/libyaml.rb]: forgot the hash. retested with a fresh build after deleting the source archive to ensure it didn't complain.

Comment (mistydemeo) [reply Library/Formula/libyaml.rb]: Thank you! I'll get this done later tonight.

**Review by mistydemeo — APPROVED** [review]

**Pull request merged**